### ISSUE TITLE
WebDriver "fuzzy" locator may lead to "unknown error: script returns unexpected result" with ChromeDriver v76

### ISSUE THREAD

**Issue by reinholdfuereder** [Codeception/Codeception#5654]
While this problem will be hopefully solved by ChromeDriver, it also shows a possible workaround and may help other Codeception users stumbling over this test failure symptom...

On the other hand maybe the "fuzzy" locator implementation can be changed/improved?

#### What are you trying to achieve?
Some Codeception WebDriver acceptance tests using "fuzzy" locators (https://codeception.com/docs/modules/WebDriver#Locating-Elements) stopped working after upgrading to ChromeDriver 76.0.3809.68 (via https://github.com/SeleniumHQ/docker-selenium/releases/tag/3.141.59-titanium) with the same symptom as https://bugs.chromium.org/p/chromedriver/issues/detail?id=3084:
```unknown error: script returns unexpected result
  (Session info: headless chrome=76.0.3809.87)
  (Driver info: chromedriver=76.0.3809.68 (420c9498db8ce8fcd190a954d51297672c1515d5-refs/branch-heads/3809@{#864}),platform=Linux 4.15.0-55-generic x86_64)
```

It worked before with older ChromeDriver versions (v71-75), but also fails (on my local desktop environment) with ChromeDriver v76 in combination with Chrome v75 (before updating it to v76), but works fine with ChromeDriver v75 in combination with Chrome v76.

The problem is e.g. within Codeceptions acceptance tester step "fillField(<FIELD LOCATOR STRING>, '...')", which in turn does a "findField(<FIELD LOCATOR STRING>)", which in turn tries to use some semi-magic that seemingly stopped working now (with ChromeDriver v76): the magic is trying to use the <FIELD LOCATOR STRING> with different locator types for maximum convenience for the test author, so e.g. it tries to use an XPath expression with it to find the field (although it is an CSS locator expression)...

Concrete example:
```
  public function withXSitesAndFuzzyLocatorFails(\AcceptanceTesterWebDriver $I) {
    $I->amOnUrl('https://www.xsites.online/redx');
    $I->fillField('.login-form #ses_id', 'my name');
  }
```

Like in the other comments from https://bugs.chromium.org/p/chromedriver/issues/detail?id=3084 there is JS code using "prototype" method in the website.

It works fine however, when using WebDriver FindElements with CSS Selector instead:
```
  public function withXSitesAndCssSelectorWorks(\AcceptanceTesterWebDriver $I) {
    $I->amOnUrl('https://www.xsites.online/redx');
    $I->fillField([ 'css' => '.login-form #ses_id' ], 'my name');
  }
```

### Details

* Codeception version: 3.0.3
* PHP Version: 7.2
* Operating System: Linux
* Installation type: Phar
* ChromeDriver: 76

**Comment by reinholdfuereder** [comment]
Please note that the ChromeDriver bug was confirmed in https://bugs.chromium.org/p/chromedriver/issues/detail?id=3084#c16 and fixed in the meantime and also "back ported to versions 76 and 77".

Unfortunately there was no new release of https://github.com/SeleniumHQ/docker-selenium/releases yet -- even my polite ticket https://github.com/SeleniumHQ/docker-selenium/issues/960 was declined ;-)

**Comment by reinholdfuereder** [comment]
With a custom built Selenium Docker Image with fixed ChromeDriver, namely `browser 'chrome' v76.0.3809.132 with ChromeDriver version '76.0.3809.126 (d80a294506b4c9d18015e755cee48f953ddc3f2f-refs/branch-heads/3809@{#1024})' running on platform 'Linux'` I can confirm the/my problem is fixed :-)

=> closing this issue
